index.html

```diff
@@ -0,0 +1,272 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Weather Dashboard</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Dancing+Script:wght@400;600;700&family=Josefin+Sans:wght@400;500;700&family=Lato:wght@100;400;700&family=Poppins:wght@300;400;700&family=Roboto+Condensed:wght@300;400;700&family=Roboto:wght@400;500;700&family=Smokum&family=Source+Sans+Pro:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="./assets/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="navbar navbar-dark bg-dark" class="navbar">
+        <h1 class="centre" id="title-page">
+          <a href="./index.html" class="box"
+            ><i class="fas fa-cloud-sun"></i> Weather Dashboard
+            <i class="fas fa-cloud-moon-rain"></i
+          ></a>
+        </h1>
+      </nav>
+    </header>
+
+    <main>
+      <div class="row g-0">
+        <!-- form input to search for cities and previous searches-->
+        <div class="col-sm-12 col-md-3">
+          <!-- <div>
+            <h2 class="mt-3 text-center">Search for a city</h2>
+            <hr />
+            <form class="p-3" id="search-form">
+              <div class="mb-3">
+                <input
+                  type="email"
+                  class="form-control"
+                  id="search-box"
+                  aria-describedby="emailHelp"
+                  placeholder="Enter city name"
+                />
+              </div>
+              <div class="d-grid gap-2">
+                <button class="btn btn-primary" type="button">Search</button>
+              </div>
+            </form>
+          </div> -->
+
+          <!-- col for form and recent searches -->
```

Comment:
If code isn't being used, please remove it rather than leave it commented out.

Reply:
Thank you i will remove it